_layouts/home.html

```diff
@@ -23,6 +23,12 @@ <h3>
         </h3>
         {%- if site.show_excerpts -%}
           {{ post.excerpt }}
+          {% assign content_words = post.content | number_of_words %}
+          {% assign excerpt_words = post.excerpt | number_of_words %}
+
+          {% if content_words != excerpt_words  %}
+            <p><a href="{{ post.url  | relative_url }}">Read more</a></p>
```

Comment:
This introduces an a11y issue when all read more links will have the same label. People using Voice over or similar software, will not be able to distinguish where those links get to.

Reply:
I updated per your suggestion. Since the whole post is clickable, I have to strip the html from the excerpt too.